src/models/user.js

```diff
@@ -0,0 +1,31 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db.js');
+
+const User = sequelize.define('User', {
+  id: {
+    type: DataTypes.UUID,
+    primaryKey: true,
+    defaultValue: DataTypes.UUIDV4,
+  },
+  name: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  email: {
+    type: DataTypes.STRING,
+    allowNull: false,
+    unique: true,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
```

Comment:
Ensure that the password is hashed before being stored in the database to protect user credentials. This should be handled in the service layer or through Sequelize hooks.